openreview/conference/templates/areachairWebfield.js

```diff
@@ -612,6 +635,62 @@ var renderTableRows = function(rows, container) {
   $(container).append(tableHtml);
 }
 
+var postRenderTable = function(rows) {
+
+  currentRankings = [];
+  rows.forEach(function(row) {
+    if(row[4].ranking) {
+      currentRankings.push(row[4].ranking.tag);
+    }
+  });
+  paperRankingInvitation.reply.content.tag['value-dropdown'] = _.difference(availableOptions, currentRankings);
+
+  rows.forEach(function(row) {
+    var noteId = row[4].noteId;
+    var noteNumber = row[4].paperNumber;
+    var paperRanking = row[4].ranking;
+    $metaReviewStatusContainer = $('#' + noteNumber + '-metareview-status');
+    if (!$metaReviewStatusContainer.length) {
+      return;
+    }
+
+    var $tagWidget = view.mkTagInput(
+      'tag',
+      paperRankingInvitation && paperRankingInvitation.reply.content.tag,
+      paperRanking ? [paperRanking] : [],
+      {
+        forum: noteId,
+        placeholder: (paperRankingInvitation && paperRankingInvitation.reply.content.tag.description) || (paperRankingInvitation && prettyId(paperRankingInvitation.id)),
+        label: paperRankingInvitation && view.prettyInvitationId(paperRankingInvitation.id),
+        readOnly: false,
+        onChange: function(id, value, deleted, done) {
+          var body = {
+            id: id,
+            tag: value,
+            signatures: [user.profile.id],
+            readers: [CONFERENCE_ID],
+            forum: noteId,
+            invitation: paperRankingInvitation.id,
+            ddate: deleted ? Date.now() : null
+          };
+          body = view.getCopiedValues(body, paperRankingInvitation.reply);
+          Webfield.post('/tags', body)
+          .then(function(result) {
+            row[4].ranking = result; //not sure if this is the best way to do it
+            postRenderTable(rows);
+            done(result);
+          })
+          .fail(function(error) {
+            promptError(error ? error : 'The specified tag could not be updated');
+          });
+        }
+      }
+    );
+    $('.tag-widget', '#' + noteNumber + '-metareview-status').remove();
```

Comment:
I think this might be more clear if you did `$metaReviewStatusContainer.find('.tag-widget').remove()`